### PR TITLE
Use os sdk as dependency

### PR DESCRIPTION
**Pull request by tmathern**
Use open source sdk

## Description
Use open source sdk, `@adobe/asset-compute-sdk`

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: generators/add-action/asset-compute/index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class AssetComputeGenerator extends ActionGenerator {
     this.addAction(this.props.actionName, './_worker.js', {
       tplContext: this.props,
       dependencies: {
-        '@nui/library': '^19.0.0' // will be replaced with open sourced @adobe scope
+        '@adobe/asset-compute-sdk': '^1.0.2' // will be replaced with open sourced @adobe scope
```
Comment (jdelbick) [inline generators/add-action/asset-compute/index.js:36]: remove comment now that it is replaced

File: generators/add-action/asset-compute/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ class AssetComputeGenerator extends ActionGenerator {
...
       },
       devDependencies: {
         '@adobe/eslint-config-asset-compute': '^1.0.0'
```
Comment (jdelbick) [inline generators/add-action/asset-compute/index.js:39]: update eslint version here^

**Review by jdelbick — APPROVED** [review]
Approving assuming you tried this out locally to make sure this didn't break anything.

Btw, when i worked on this, to test locally I made changes to `aio-cli-plugin-app` to include our generator, and run it using `bin/run`

**Comment by tmathern** [comment]
Closing in favor of https://github.com/adobe/generator-aio-app/pull/55, that I'll merge into this fork after it's merged in upstream.

File: generators/add-action/asset-compute/index.js (hunk 2 above)
Comment (tmathern) [reply generators/add-action/asset-compute/index.js:39]: done.

File: generators/add-action/asset-compute/index.js (hunk 1 above)
Comment (tmathern) [reply generators/add-action/asset-compute/index.js:36]: done.

**Pull request closed without merging**